packages/preview/chi-cv/0.1.0/typst.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "chi-cv"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["@skyzh"]
+license = "CC0-1.0"
+description = "A minimal and fully-customizable CV template."
+repository = "https://github.com/skyzh/chicv"
+keywords = ["cv"]
+categories = ["cv"]
+
+[template]
+path = "template"
+entrypoint = "cv.typ"
+thumbnail = "cv.png"
```

Comment:
This won't work: the thumbnail path is relative to the root of the package, not to the template directory. Can you please move the `cv.png` file outside of this directory?

Reply:
sounds good, moved in https://github.com/typst/packages/pull/703/commits/c5cca8827558ad22cbe52f7dcb963248591a1e0d